README.md

```diff
@@ -93,21 +91,69 @@ The loopback4-billing package relies on the configuration of the chosen billing
 
 ### Step 1: Set Up Billing Provider
 
-#### ChargeBee -
-
+#### For ChargeBee -
 To use Chargebee as the billing provider, you need to configure the Chargebee API keys and site URL in your application. You can set these values in the environment variables of your LoopBack 4 project.
 
 ```
 CHARGEBEE_API_KEY=your_chargebee_api_key
```

Comment:
What is the use of this variable.. I do not see this variable used below any where.